node/network/bridge/src/lib.rs

```diff
@@ -256,8 +272,8 @@ fn action_from_overseer_message(
 				=> Action::SendValidationMessage(peers, msg),
 			NetworkBridgeMessage::SendCollationMessage(peers, msg)
 				=> Action::SendCollationMessage(peers, msg),
-			NetworkBridgeMessage::ConnectToValidators(peer_set, validators, res)
-				=> Action::ConnectToValidators(peer_set, validators, res),
+			NetworkBridgeMessage::ConnectToAuthorities(peer_set, authorities, res)
```

Comment:
It should be called `ConnectToValidators`. `Authority` is a term used lower down in the stack

Reply:
Also I'd suggest something like `SetConnectedValidators`. It's a state change that we perform, not a one-shot action.

Reply:
ATM we call `set_priority_group`, but what we actually want is `add_to_priority_group`, so it's an incremental change, see https://github.com/paritytech/polkadot/pull/1699#discussion_r486389522.

There is an issue with an unbounded growth of priority_group though, it's not clear to how it should handled https://github.com/paritytech/polkadot/pull/1699#issuecomment-690966735.